tests/test_merge.py

```diff
@@ -37,10 +39,10 @@ def test_merge_from_fixtures(tmp_path):
     '''
     Assert that general_merge merges subnets correctly.
     '''
-
-    base_nets, base_subnet_count = __read_test_vectors("tests/data/base_file.csv")
+    data_dir = os.path.join(os.path.dirname(__file__), "data")
```

Comment:
nit: Instead of `os.path.join` I would prefer if you would continue using Pathlib and the `/` operator like in the rest of the code here where paths are concatenated.